gym-unity/gym_unity/envs/__init__.py

```diff
@@ -226,21 +236,31 @@ def _single_step(self, info):
         else:
             default_observation = self._get_vector_obs(info)[0, :]
 
-        return (default_observation, info.reward[0], info.done[0], info)
+        return (
+            default_observation,
+            info.reward[0],
+            info.done[0],
+            {"batched_step_result": info},
```

Comment:
Ideally we'd turn the batched_step_result NamedTuple into a dict of its fields so it can be easily accessed without knowing anything about the `BatchedStepResult` object, but this is technically in spec with Gym.

Reply:
Previously, we were using a BrainInfo as the value, so I think I'm OK with this for now. 

I'm not sure what other keys are generally used, but here's where it was failing in the original issue: https://github.com/hill-a/stable-baselines/blob/6bdb7ce8ed03224a8c03d3197ff59b86cbcb0b94/stable_baselines/td3/td3.py#L336-L343

Reply:
Previously, we were using a BrainInfo as the value, so I think I'm OK with this for now.

I'm not sure what other keys are generally used, but here's where it was failing in the original issue: https://github.com/hill-a/stable-baselines/blob/6bdb7ce8ed03224a8c03d3197ff59b86cbcb0b94/stable_baselines/td3/td3.py#L336-L343

Reply:
The keys available are completely arbitrary :P I suppose the ATARI envs have some standard ones that are available.